webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -206,6 +206,7 @@ def bugzilla_submit_url(report, parsed_dump, crashing_thread, bug_product):
 
     comment = render_to_string('crashstats/bugzilla_comment.txt', {
         'uuid': report['uuid'],
+        'java_stack_trace': report.get('java_stack_trace', None),
```

Comment:
The `java_stack_trace` is just a big string, so we don't need to do any further processing/adjusting of it.